.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+## Pull Request Title
```

Comment:
@KatunaNorbert as far as I can tell from the docs, it's either
`PULL_REQUEST_TEMPLATE/` as a directory
or `pull_request_template` (non-capitals) as a filename.

Can you please double check if it works?

Reply:
ref https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

Reply:
I see, well I guess the only way to verify is to merge the PR, or manually add the file to the repo.
It might work both ways but I could make it lower case to be sure.

Reply:
tried to make it lower case but looks like it's not noticed as a change so it's not submitted

Reply:
deleted the file and added it back to change the name